lib/store/src/StoriesListStore.ts

```diff
@@ -14,29 +14,38 @@ export class StoriesListStore {
     this.fetchStoriesList = fetchStoriesList;
   }
 
-  async initialize() {
-    return this.cacheStoriesList();
-  }
+  initialize(sync: false): Promise<void>;
+
+  initialize(sync: true): void;
 
-  initializeSync() {
-    return this.cacheStoriesListSync();
+  initialize(sync: boolean) {
+    return sync ? this.cacheStoriesList(true) : this.cacheStoriesList(false);
```

Comment:
It's kind of weird you can't just do
```suggestion
    return this.cacheStoriesList(sync);
```

But TS wants this.

Reply:
Possibly because `sync` can be `undefined`. Would it work if you cast it to a boolean (`!!sync`)?

Reply:
Nah the error is: 

<img width="717" alt="image" src="https://user-images.githubusercontent.com/132554/132618591-7e514f76-5d0d-4cb2-8307-57bcb74b7fb8.png">

I think you would have to "cast" it to `true` or `false` which is kind of crazy.